### PR TITLE
Clarify documentation for character count

### PR DESCRIPTION
**Pull request by devonzuegel**
Thank you for making this plugin, I love it!

The command `\wordcount char` wasn't working for me, so I dug into the code and realized that I could just put `{{renderer :wordcountchar_RANDOM-ID}}` directly, and that worked. So I updated the docs in case anyone else wants to use that method too.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,53 +2,48 @@
...
+<img width="80%" src="screenshots/wordcount.gif" />
 
-# Usage
+## Usage
 
-Use `/wordcount`. It will then count the number of words below this parent block. This supports an unlimited number of nested blocks.
+| command | autocompletes to | what it does |
+| :- | :- | :- |
+| `/wordcount` | `{{renderer :wordcount_RANDOM-ID}}` | count the number of **words** below this parent block |
+| `/wordcount char` | `{{renderer :wordcountchar_RANDOM-ID}}` | count the number of **characters** below this parent block |
```
Comment (devonzuegel) [inline README.md:14]: This is the primary change in this PR. Everything else is just tiny cosmetic things to make it a little more readable.

**Comment by hkgnp** [comment]
@devonzuegel!
Thanks for your PR! Could I clarify which part wasn't working? Usually for plugins that register new slash commands, you have to restart Logseq in order to see them.

I've tested it on my end, and the command `/wordcount char` seems to work. I noticed that in your writeup, you used a backslash instead.

demo

**Comment by devonzuegel** [comment]
Oh silly me, I hadn't restarted 🙈 Sorry about that! You're right, it is working.

Ah that's because in my config.edn, [I've customized it](https://github.com/logseq/logseq/pull/3001) use `\` for my commands rather than `/`. That wasn't the source of the problem (it was that I hadn't restarted), but good eye!

Would you be open to a PR renaming `/wordcount char` → `/character count`? I think it makes more sense than having "word" in there, but it's ultimately your call of course as the maintainer!

**Comment by hkgnp** [comment]
Great to hear!

Actually, you made a very good point. Not sure what I was thinking. I've already made the change and it's in v6 in the marketplace now!

**Pull request closed without merging**